Could not compile "breez-sdk-core"

Hi, when trying to use the SDK in Rust, it fails to compile whether I try to use the `main` branch or the `0.2.7` rev as such. Probably I'm doing something wrong.

```rust
[dependencies]
...
breez-sdk-core = { git = "https://github.com/breez/breez-sdk", branch = "main" } // or
breez-sdk-core = { git = "https://github.com/breez/breez-sdk", rev = "0.2.7" }
```

## Error when using `branch ="main"`

```
error[E0432]: unresolved imports `ecies::utils::aes_decrypt`, `ecies::utils::aes_encrypt`
 --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/f5b24c7/libs/sdk-core/src/backup.rs:9:20
  |
9 | use ecies::utils::{aes_decrypt, aes_encrypt};
  |                    ^^^^^^^^^^^  ^^^^^^^^^^^ no `aes_encrypt` in `utils`
  |                    |
  |                    no `aes_decrypt` in `utils`

error[E0432]: unresolved imports `ecies::utils::aes_decrypt`, `ecies::utils::aes_encrypt`
  --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/f5b24c7/libs/sdk-core/src/greenlight/node_api.rs:16:20
   |
16 | use ecies::utils::{aes_decrypt, aes_encrypt};
   |                    ^^^^^^^^^^^  ^^^^^^^^^^^ no `aes_encrypt` in `utils`
   |                    |
   |                    no `aes_decrypt` in `utils`

error[E0277]: the trait bound `SecpError: std::error::Error` is not satisfied
 --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/f5b24c7/libs/sdk-core/src/crypt.rs:6:29
  |
6 |         Err(err) => Err(err.into()),
  |                             ^^^^ the trait `std::error::Error` is not implemented for `SecpError`
  |
  = note: required for `anyhow::Error` to implement `From<SecpError>`
  = note: required for `SecpError` to implement `Into<anyhow::Error>`

error[E0277]: the trait bound `SecpError: std::error::Error` is not satisfied
  --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/f5b24c7/libs/sdk-core/src/crypt.rs:14:29
   |
14 |         Err(err) => Err(err.into()),
   |                             ^^^^ the trait `std::error::Error` is not implemented for `SecpError`
   |
   = note: required for `anyhow::Error` to implement `From<SecpError>`
   = note: required for `SecpError` to implement `Into<anyhow::Error>`

Some errors have detailed explanations: E0277, E0432.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `breez-sdk-core` (lib) due to 4 previous errors
```


## Error when using `rev = "0.2.7"` 
```
error[E0432]: unresolved imports `ecies::utils::aes_decrypt`, `ecies::utils::aes_encrypt`
 --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/backup.rs:8:20
  |
8 | use ecies::utils::{aes_decrypt, aes_encrypt};
  |                    ^^^^^^^^^^^  ^^^^^^^^^^^ no `aes_encrypt` in `utils`
  |                    |
  |                    no `aes_decrypt` in `utils`

error[E0432]: unresolved imports `ecies::utils::aes_decrypt`, `ecies::utils::aes_encrypt`
  --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/greenlight/node_api.rs:13:20
   |
13 | use ecies::utils::{aes_decrypt, aes_encrypt};
   |                    ^^^^^^^^^^^  ^^^^^^^^^^^ no `aes_encrypt` in `utils`
   |                    |
   |                    no `aes_decrypt` in `utils`

error[E0277]: the trait bound `models::NodeState: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:145:33
    |
145 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::NodeState`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `SignMessageResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:165:33
    |
165 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `SignMessageResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `CheckMessageResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:181:33
    |
181 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `CheckMessageResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::Config: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:212:33
    |
212 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::Config`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::StaticBackupResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:230:33
    |
230 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::StaticBackupResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `BreezEvent: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:249:72
    |
249 |         move || move |task_callback| breez_events_stream(task_callback.stream_sink()),
    |                                                                        ^^^^^^^^^^^ the trait `IntoIntoDart<_>` is not implemented for `BreezEvent`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `TaskCallback::stream_sink`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/rust2dart.rs:80:12
    |
78  |     pub fn stream_sink<T, D>(&self) -> StreamSink<T>
    |            ----------- required by a bound in this associated function
79  |     where
80  |         T: IntoIntoDart<D>,
    |            ^^^^^^^^^^^^^^^ required by this bound in `TaskCallback::stream_sink`

error[E0277]: the trait bound `models::LogEntry: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:259:69
    |
259 |         move || move |task_callback| breez_log_stream(task_callback.stream_sink()),
    |                                                                     ^^^^^^^^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::LogEntry`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `TaskCallback::stream_sink`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/rust2dart.rs:80:12
    |
78  |     pub fn stream_sink<T, D>(&self) -> StreamSink<T>
    |            ----------- required by a bound in this associated function
79  |     where
80  |         T: IntoIntoDart<D>,
    |            ^^^^^^^^^^^^^^^ required by this bound in `TaskCallback::stream_sink`

error[E0277]: the trait bound `lsp::LspInformation: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:263:33
    |
263 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `lsp::LspInformation`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
    = note: required for `Vec<lsp::LspInformation>` to implement `IntoIntoDart<Vec<_>>`
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `lsp::LspInformation: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:296:33
    |
296 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `lsp::LspInformation`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
    = note: required for `std::option::Option<lsp::LspInformation>` to implement `IntoIntoDart<std::option::Option<_>>`
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `lsp::LspInformation: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:309:33
    |
309 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `lsp::LspInformation`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::BackupStatus: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:339:33
    |
339 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::BackupStatus`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `LNInvoice: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:349:33
    |
349 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `LNInvoice`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `InputType: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:362:33
    |
362 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `InputType`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::Payment: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:378:33
    |
378 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::Payment`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
    = note: required for `Vec<models::Payment>` to implement `IntoIntoDart<Vec<_>>`
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::Payment: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:391:33
    |
391 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::Payment`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
    = note: required for `std::option::Option<models::Payment>` to implement `IntoIntoDart<std::option::Option<_>>`
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::SendPaymentResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:404:33
    |
404 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::SendPaymentResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::SendPaymentResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:420:33
    |
420 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::SendPaymentResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::ReceivePaymentResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:436:33
    |
436 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::ReceivePaymentResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `LnUrlPayResult: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:449:33
    |
449 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `LnUrlPayResult`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::LnUrlWithdrawResult: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:465:33
    |
465 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::LnUrlWithdrawResult`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::LnUrlCallbackStatus: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:481:33
    |
481 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::LnUrlCallbackStatus`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `fiat::Rate: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:494:33
    |
494 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `fiat::Rate`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
    = note: required for `Vec<fiat::Rate>` to implement `IntoIntoDart<Vec<_>>`
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `FiatCurrency: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:504:33
    |
504 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `FiatCurrency`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
    = note: required for `Vec<FiatCurrency>` to implement `IntoIntoDart<Vec<_>>`
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::SendOnchainResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:514:33
    |
514 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::SendOnchainResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::SwapInfo: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:530:33
    |
530 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::SwapInfo`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::BuyBitcoinResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:543:33
    |
543 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::BuyBitcoinResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::SweepResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:556:33
    |
556 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::SweepResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::SwapInfo: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:569:33
    |
569 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::SwapInfo`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
    = note: required for `Vec<models::SwapInfo>` to implement `IntoIntoDart<Vec<_>>`
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::RefundResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:579:33
    |
579 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::RefundResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::SwapInfo: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:592:33
    |
592 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::SwapInfo`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
    = note: required for `std::option::Option<models::SwapInfo>` to implement `IntoIntoDart<std::option::Option<_>>`
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::ReverseSwapInfo: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:602:33
    |
602 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::ReverseSwapInfo`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
    = note: required for `Vec<models::ReverseSwapInfo>` to implement `IntoIntoDart<Vec<_>>`
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::OpenChannelFeeResponse: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:615:33
    |
615 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::OpenChannelFeeResponse`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `models::ReverseSwapPairInfo: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:631:33
    |
631 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `models::ReverseSwapPairInfo`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `RecommendedFees: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/bridge_generated.rs:644:33
    |
644 |     FLUTTER_RUST_BRIDGE_HANDLER.wrap(
    |                                 ^^^^ the trait `IntoIntoDart<_>` is not implemented for `RecommendedFees`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `wrap`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/handler.rs:51:18
    |
47  |     fn wrap<PrepareFn, TaskFn, TaskRet, D>(&self, wrap_info: WrapInfo, prepare: PrepareFn)
    |        ---- required by a bound in this associated function
...
51  |         TaskRet: IntoIntoDart<D>,
    |                  ^^^^^^^^^^^^^^^ required by this bound in `Handler::wrap`

error[E0277]: the trait bound `BreezEvent: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/binding.rs:375:20
    |
375 |             stream.add(e);
    |                    ^^^ the trait `IntoIntoDart<_>` is not implemented for `BreezEvent`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `StreamSink::<T>::add`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/rust2dart.rs:139:12
    |
137 |     pub fn add<D: IntoDart>(&self, value: T) -> bool
    |            --- required by a bound in this associated function
138 |     where
139 |         T: IntoIntoDart<D>,
    |            ^^^^^^^^^^^^^^^ required by this bound in `StreamSink::<T>::add`

error[E0277]: the trait bound `models::LogEntry: IntoIntoDart<_>` is not satisfied
   --> /Users/darioanongbavarela/.cargo/git/checkouts/breez-sdk-d2a5b0f6a3ca00f6/7e5f4f0/libs/sdk-core/src/binding.rs:399:29
    |
399 |             self.log_stream.add(LogEntry {
    |                             ^^^ the trait `IntoIntoDart<_>` is not implemented for `models::LogEntry`
    |
    = help: the following other types implement trait `IntoIntoDart<D>`:
              <bool as IntoIntoDart<bool>>
              <i8 as IntoIntoDart<i8>>
              <i16 as IntoIntoDart<i16>>
              <i32 as IntoIntoDart<i32>>
              <i64 as IntoIntoDart<i64>>
              <usize as IntoIntoDart<usize>>
              <u8 as IntoIntoDart<u8>>
              <u16 as IntoIntoDart<u16>>
            and 22 others
note: required by a bound in `StreamSink::<T>::add`
   --> /Users/darioanongbavarela/.cargo/registry/src/index.crates.io-6f17d22bba15001f/flutter_rust_bridge-1.80.1/src/rust2dart.rs:139:12
    |
137 |     pub fn add<D: IntoDart>(&self, value: T) -> bool
    |            --- required by a bound in this associated function
138 |     where
139 |         T: IntoIntoDart<D>,
    |            ^^^^^^^^^^^^^^^ required by this bound in `StreamSink::<T>::add`

Some errors have detailed explanations: E0277, E0432.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `breez-sdk-core` (lib) due to 38 previous errors
```

--- a/libs/sdk-core/Cargo.toml
+++ b/libs/sdk-core/Cargo.toml
@@ -22,7 +22,7 @@ gl-client = { git = "https://github.com/Blockstream/greenlight.git", features = 
 zbase32 = "0.1.2"
 base64 = "0.13.0"
 chrono = "0.4"
-ecies = { version = "0.2", default-features = false, features = ["pure"] }
+ecies = { version = "0.2.6", default-features = false, features = ["pure"] }
 env_logger = "0.10"
 futures = "0.3.28"
 ripemd = "0.1"

--- a/libs/sdk-core/src/backup.rs
+++ b/libs/sdk-core/src/backup.rs
@@ -6,7 +6,7 @@ use crate::{
 };
 
 use anyhow::{anyhow, Result};
-use ecies::utils::{aes_decrypt, aes_encrypt};
+use ecies::symmetric::{sym_decrypt, sym_encrypt};
 use miniz_oxide::{deflate::compress_to_vec, inflate::decompress_to_vec_with_limit};
 use std::{
     fs::{self, File},
@@ -390,11 +390,11 @@ impl BackupWorker {
         match state {
             Some(state) => {
                 let mut decrypted =
-                    aes_decrypt(self.encryption_key.as_slice(), state.data.as_slice());
+                    sym_decrypt(self.encryption_key.as_slice(), state.data.as_slice());
                 if decrypted.is_none() {
                     warn!("Failed to decrypt backup with new key, trying legacy key");
                     decrypted =
-                        aes_decrypt(self.legacy_encryption_key.as_slice(), state.data.as_slice());
+                        sym_decrypt(self.legacy_encryption_key.as_slice(), state.data.as_slice());
                 }
                 let decrypted_data = decrypted.ok_or(anyhow!("Failed to decrypt backup"))?;
                 match decompress_to_vec_with_limit(&decrypted_data, 4000000) {
@@ -419,7 +419,7 @@ impl BackupWorker {
             compressed_data.len()
         );
         let encrypted_data =
-            aes_encrypt(self.encryption_key.as_slice(), compressed_data.as_slice())
+            sym_encrypt(self.encryption_key.as_slice(), compressed_data.as_slice())
                 .ok_or(anyhow!("Failed to encrypt backup"))?;
         let version = self.inner.push(version, encrypted_data.clone()).await?;
         Ok((version, encrypted_data))

--- a/libs/sdk-core/src/crypt.rs
+++ b/libs/sdk-core/src/crypt.rs
@@ -1,9 +1,9 @@
-use anyhow::Result;
+use anyhow::{anyhow, Result};
 
 pub fn encrypt(key: Vec<u8>, msg: Vec<u8>) -> Result<Vec<u8>> {
     match ecies::encrypt(key.as_slice(), msg.as_slice()) {
         Ok(res) => Ok(res),
-        Err(err) => Err(err.into()),
+        Err(err) => Err(anyhow!(err.to_string())),
     }
 }
 
@@ -11,6 +11,6 @@ pub fn encrypt(key: Vec<u8>, msg: Vec<u8>) -> Result<Vec<u8>> {
 pub fn decrypt(key: Vec<u8>, msg: Vec<u8>) -> Result<Vec<u8>> {
     match ecies::decrypt(key.as_slice(), msg.as_slice()) {
         Ok(res) => Ok(res),
-        Err(err) => Err(err.into()),
+        Err(err) => Err(anyhow!(err.to_string())),
     }
 }

--- a/libs/sdk-core/src/greenlight/node_api.rs
+++ b/libs/sdk-core/src/greenlight/node_api.rs
@@ -13,7 +13,7 @@ use bitcoin::secp256k1::PublicKey;
 use bitcoin::secp256k1::Secp256k1;
 use bitcoin::util::bip32::{ChildNumber, ExtendedPrivKey};
 use bitcoin::{Address, OutPoint, Script, Sequence, Transaction, TxIn, TxOut, Txid, Witness};
-use ecies::utils::{aes_decrypt, aes_encrypt};
+use ecies::symmetric::{sym_decrypt, sym_encrypt};
 use gl_client::node::ClnClient;
 use gl_client::pb::cln::listinvoices_invoices::ListinvoicesInvoicesStatus;
 use gl_client::pb::cln::listpays_pays::ListpaysPaysStatus;
@@ -99,11 +99,11 @@ impl Greenlight {
         let parsed_credentials: Result<GreenlightCredentials> = match credentials {
             // In case we found existing credentials, try to decrypt them and connect to the node
             Some(creds) => {
-                let mut decrypted_credentials = aes_decrypt(encryption_key_slice, creds.as_slice());
+                let mut decrypted_credentials = sym_decrypt(encryption_key_slice, creds.as_slice());
                 if decrypted_credentials.is_none() {
                     info!("Failed to decrypt credentials, trying legacy key");
                     decrypted_credentials =
-                        aes_decrypt(legacy_encryption_key_slice, creds.as_slice());
+                        sym_decrypt(legacy_encryption_key_slice, creds.as_slice());
                 }
                 match decrypted_credentials {
                     Some(creds) => {
@@ -145,7 +145,7 @@ impl Greenlight {
         let res = match parsed_credentials {
             Ok(creds) => {
                 let json_creds = serde_json::to_string(&creds)?.as_bytes().to_vec();
-                let encryptd_creds = aes_encrypt(encryption_key_slice, json_creds.as_slice());
+                let encryptd_creds = sym_encrypt(encryption_key_slice, json_creds.as_slice());
                 match encryptd_creds {
                     Some(c) => {
                         persister.set_gl_credentials(c)?;

